features/support/env.rb

```diff
@@ -1,4 +1,7 @@
 require 'cucumber/rails'
+require 'coveralls'
+Coveralls.wear_merged!('rails')
```

Comment:
these two lines need to be on top of the file

Reply:
Done.